### ISSUE TITLE
Mockup

### ISSUE THREAD

**Issue by CadmiumRed40** [CadmiumRed40/FindAFlick#13]

**Comment by CadmiumRed40** [comment]
https://www.figma.com/file/iBfbygtjJ7bWh68UBecXq6/Untitled?node-id=0%3A1

**Comment by CadmiumRed40** [comment]
![Find A Flick](https://user-images.githubusercontent.com/80559746/188102456-5721684f-d52e-4ae1-8876-c9becdbeeacc.png)

**Comment by floorrinse** [comment]
@CadmiumRed40 Can we also get a design for the index page? (the `/` page that has "Create your very own movie & tv show list!"). The index page should serve as a home page to let users what this app is. It can be simple with the same h1 but with the gray background color as the above images with `Login` and `Signup` buttons. 

There can also be a paragraph blurb here to tell users what this app is. You don't need to work on the paragraph wording, maybe we can ticket that for later.

**Comment by floorrinse** [comment]
@CadmiumRed40 Looking at the `Dashboard (while searching)` mockup, I don't think we need the page to display movie poster img results as user is typing. That seems like a bit more work that could be considered an enhancement for later. We can have the page be static until user presses enter, or clicks "Find a Flick". 

I think we should move the "Find a Flick" button to the side of the searchbar if we want the text results to populate as user is typing though.

What are your thoughts?

**Comment by CadmiumRed40** [comment]
@floorrinse I agree! Should the search bar be flush with the logo, profile icon/user name and be persistent on every page? I'll apply these changes by tomorrow and update the sheet. Live update will be changed to a feature enhancement.

**Comment by svmarinez** [comment]
<img width="1440" alt="Screenshot 2022-09-03 at 20 39 00" src="https://user-images.githubusercontent.com/30931982/188283996-90243f4d-9d2a-4eb4-b0d6-2476f4606fe8.png">
![FindAFlick_copy](https://user-images.githubusercontent.com/30931982/188284016-663bf04e-29ce-47cd-8ac3-b933fa4cc0f9.jpeg)

WIP

**Comment by floorrinse** [comment]
> <img alt="Screenshot 2022-09-03 at 20 39 00" width="1440" src="https://user-images.githubusercontent.com/30931982/188283996-90243f4d-9d2a-4eb4-b0d6-2476f4606fe8.png">
> 
> ![FindAFlick_copy](https://user-images.githubusercontent.com/30931982/188284016-663bf04e-29ce-47cd-8ac3-b933fa4cc0f9.jpeg)
> WIP

Woo this looks so good!! Nice work Sara! 🥳

**Comment by floorrinse** [comment]
> @floorrinse I agree! Should the search bar be flush with the logo, profile icon/user name and be persistent on every page? I'll apply these changes by tomorrow and update the sheet. Live update will be changed to a feature enhancement.

I don't think the search bar needs to be flush with the other elements, esp if it's a text input field but open to more discussion. 

I'm looking at Netflix as a reference and the search bar doesn't show up as a text input field at first so we could do something similiar down the road 👍

**Comment by CadmiumRed40** [comment]
![Find A Flick](https://user-images.githubusercontent.com/80559746/188299596-5dad83e7-372b-4531-bd29-1e6e6d9911f6.png)

**Comment by AllieGie** [comment]
@CadmiumRed40 to clarify, the signup form is meant to be one form? At first glance, I thought they were two separate forms. After looking at the buttons, realized it's meant to be one? I have them coded as two and can make the changes once you confirm. Thanks!

**Comment by CadmiumRed40** [comment]
@AllieGie yes it's one form.

**Comment by floorrinse** [comment]
@CadmiumRed40 would it make sense to just have them as two forms? I think we've already started coding it that way but I could be wrong (based on @svmarinez work). My initial thought was that the `/` page would have 2 buttons where one goes to `/login` (the first form on the top of the design) and another goes to `/signup` (the second form on the bottom of the design). I just wanna make sure we're all the same page (I'll post this comment up in Discord too).

Since we're really close to the deadline, I'm good with whatever works best and easiest!

**Comment by CadmiumRed40** [comment]
@floorrinse I imagined it as 2 sperate forms but honestly whatever is easiest.
